src/PlcNodeManager.cs

```diff
@@ -117,6 +117,46 @@ private void UpdateNodes()
             }
         }
 
+#pragma warning disable IDE0060 // Remove unused parameter
+        public void UpdateEventInstances(object state, ElapsedEventArgs elapsedEventArgs)
+#pragma warning restore IDE0060 // Remove unused parameter
+        {
+            UpdateEventInstances();
+        }
+
+#pragma warning disable IDE0060 // Remove unused parameter
+        public void UpdateVeryFastEventInstances(object state, FastTimerElapsedEventArgs elapsedEventArgs)
+#pragma warning restore IDE0060 // Remove unused parameter
+        {
+            UpdateEventInstances();
+        }
+
+        private void UpdateEventInstances()
+        {
+            uint eventInstanceCycle = _eventInstanceCycle++;
+
+            for (uint i=0; i<PlcSimulation.EventInstanceCount; i++)

```

Comment:
Please apply standard VS formatting to the file(s). Otherwise there may be many changes to the file the next time somebody formats it(them).

Reply:
Good point, done.